src/components/ArticleCard.jsx

```diff
@@ -3,12 +3,23 @@ import "../styling/articlecard.css"
 
 function ArticleCard({article}) {
 
+  const dateObject = new Date(article.created_at);
+
+  const formattedDate = dateObject.toLocaleString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    timeZoneName: 'short',
+  });
+
   return (
     <>
       <div className="article-card">
       <div className="author">{article.author}</div>
-      <div className="title">{article.title}</div>
-      <div className="topic">{article.topic}</div>
+      <div className="title"> {article.title}</div>
```

Comment:
See this for more interesting ways on displaying information: https://www.refactoringui.com/previews/labels-are-a-last-resort